packages/rollup-plugin/runtime.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-var */
+import { stylesheet } from '@stylable/runtime/esm/runtime';
```

Comment:
We only need to stylesheet part and not any other exports from runtime.

It was JS because we used to run tests from source and we didn't want to use ts plugin for rollup.

Reply:
Correction. it was also because we transpile to cjs and rollup does not accept it be default.